packages/@ember/-internals/metal/tests/computed_test.js

```diff
@@ -962,15 +1027,26 @@ moduleFor(
 
       assert.equal(get(obj, 'plusOne'), 6);
       assert.equal(plusOneDidChange, 2);
+
+      destroy(obj);
+      await runLoopSettled();
     }
   }
 );
 
 moduleFor(
   'computed - default setter',
   class extends AbstractTestCase {
+    afterEach() {
+      if (obj !== undefined) {
+        destroy(obj);
+        obj = undefined;
+      }
+      return runLoopSettled();
+    }
```

Comment:
Might have been easier to make a custom class in this file:

```js
let obj;
class ObjCleaningTestCase extends AbstractTestCase {
  afterEach() {
    if (obj !== undefined) {
      destroy(obj);
      obj = undefined;
      return runLoopSettled();
    }
  }
}
```

And updated the `moduleFor`'s use it instead of `AbstractTestCase`